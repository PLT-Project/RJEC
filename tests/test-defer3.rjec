func foo (x int, y int) int {
    defer printi(x);
    defer printi(y);

    z := x * y;
    x = x + 1;
    printi(x);
    return z;
}

func main() {
    x, y := 2, 8;
    defer printi(foo(x, y));

    for i := 0; i < 5; i = i + 1 {
        x = x + 1;
        y = y + 2;
    }
    printi(foo(x, y));
}