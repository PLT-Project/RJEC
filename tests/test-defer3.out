3
8
2
8
18
7
126
16